<?xml version="1.0" ?>
<launch>
	<arg name="frameID"  default = "wide_camera" />
	<arg name="frameTopic_combined"  default = "wide/image_raw" />
  <arg name="visualization" value="true" />
	<arg name="isauto" value="false" />

	<node name="stereo_record_node" pkg="flea3" type="stereo_record_node" output="screen" >

		<param name="left/Guid" value="16243862" />
    <param name="right/Guid" value="15231302" />
    <param name="cvmat_show" value="$(arg visualization)" />
		<param name="auto_shutter" value="$(arg isauto)" />
		<param name="showfrequency_ratio" value="2" />
		<!--  under visualization true: show every ratio frames! -->
    <param name="image/width" value="1280" />
    <param name="image/height" value="1024" />
    <param name="shutter_speed" value="16.4" />
    <param name="publish/frequency" value="50" />
    <param name="publish/frame_id" value="$(arg frameID)" />
    <param name="publish/topic_name" value="$(arg frameTopic_combined)" />

		<param name="publish/left_topic"            value="my_stereo/left/image_raw" />
		<param name="publish/left_frame_id"         value="wide_camera" />

		<param name="publish/right_topic"           value="my_stereo/right/image_raw" />
		<param name="publish/right_frame_id"        value="wide_camera" />

    <param name="roi/upbound" value="60" />
    <param name="roi/downbound" value="120" />

	</node>

</launch>
